Source/ACE.Server/Managers/PlayerManager.cs

```diff
@@ -206,6 +206,31 @@ public static List<Player> GetAllOnline()
             return results;
         }
 
+        /// <summary>
+        /// This will return null if the player wasn't found.
+        /// </summary>
+        public static IPlayer GetPlayer(ObjectGuid guid)
```

Comment:
The following functions are already defined that appear to do the same thing:
public static IPlayer FindByGuid(ObjectGuid guid)
public static IPlayer FindByGuid(ObjectGuid guid, out bool isOnline)
public static IPlayer FindByGuid(uint guid)
public static IPlayer FindByGuid(uint guid, out bool isOnline)

Reply:
Removed GetPlayer, redirected HandleActionDisplayPlayerConsentList() to FindByGuid